app/models/ai_persona.rb

```diff
@@ -232,9 +232,28 @@ def class_instance
     end
   end
 
+  FIRST_PERSONA_USER_ID = -1200
+
   def create_user!
     raise "User already exists" if user_id && User.exists?(user_id)
 
+    # find the first id smaller than FIRST_USER_ID that is not taken
+    id = nil
+
+    id = DB.query_single(<<~SQL, FIRST_PERSONA_USER_ID, FIRST_PERSONA_USER_ID - 200).first
```

Comment:
I guess we don't worry about this for now...but what if > 200 persona users are made :thinking: Might be good to make a site-wide persona count limit to deal with this. Anyway, not holding up this current PR :+1:

Reply:
oh there is a fallback afterwards just in case... we take minimum ... very unlikely to hit it, but it is handled.